src/datachain/lib/file.py

```diff
@@ -249,6 +265,13 @@ def export(
         dst_dir = os.path.dirname(dst)
         os.makedirs(dst_dir, exist_ok=True)
 
+        if link_type == "symlink":
+            try:
+                return self._symlink_to(dst)
+            except OSError as exc:
+                if exc.errno not in (errno.ENOTSUP, errno.EXDEV, errno.ENOSYS):
+                    raise
```

Comment:
What should we do in case of overwrite? 

Looks like `export` allows you to overwrite files over and over again, but this `_symlink_to` will fail with `FileExistsError`.